app/views/reductions.html

```diff
@@ -14,88 +14,102 @@
     <div class="message message--success">{{ successText }}</div>
     {% endif %}
 
+    {% set columnNameType='Type' %}
+    {% set columnNameHours='Hours' %}
+    {% set columnNameStartDate='Start date' %}
+    {% set columnNameEndDate='End date' %}
+
     <a href="/offender-manager/{{ linkId }}/add-reduction" class="button">Add a reduction</a>
     
-    <h2 class="heading-large">Active<span class="visuallyhidden"> reductions</span></h2>
-
+    <h2 id="headingActive" class="heading-large">Active<span class="visuallyhidden"> reductions</span></h2>
+    
     <table class="sln-table-reductions">
+        <colgroup>
+            <col span="1" style="width:35%;">
+        </colgroup>
         <thead>
             <tr>
-                <th id="active_type">Type</th>
-                <th id="active_hours">Hours</th>
-                <th id="active_start_date">Start date</th>
-                <th id="active_end_date">End date</th>
+                <th id="active_type">{{ columnNameType }}</th>
+                <th id="active_hours">{{ columnNameHours }}</th>
+                <th id="active_start_date">{{ columnNameStartDate }}</th>
+                <th id="active_end_date">{{ columnNameEndDate }}</th>
                 <th></th>
             </tr>
         </thead>
 
         <tbody class="">
             {% if newReduction %}
```

Comment:
Where is this newReduction set?

Reply:
It seem when we do a new reduction the call is redirected to this reduction page with a full query returned including the new added reduction. This might not be required as a result.